Feature/work02

NazRudy opened the pull request:
Калькулятор, урок02
Нацветов Р.И.

NazRudy committed:
дисплей и первый ряд кнопок

NazRudy committed:
второй ряд кнопок и 7

NazRudy committed:
третий ряд кнопок

NazRudy committed:
четвертый ряд кнопок

NazRudy committed:
связь с переменными

primus-anonymous commented on app/src/main/java/com/naz/calculatorhomework/MainActivity.java:
```diff
@@ -3,12 +3,32 @@
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        TextView display=findViewById(R.id.display);
```
Надо форматнуть файл, все операторы присваивания слиплись

NazRudy replied:
Форматнул

primus-anonymous commented on app/src/main/java/com/naz/calculatorhomework/MainActivity.java:
```diff
@@ -3,12 +3,32 @@
+        Button num8=findViewById(R.id.button8);
+        Button num9=findViewById(R.id.button9);
+        Button num0=findViewById(R.id.button0);
+        Button ADD=findViewById(R.id.buttonADD);
```
add, sub, mul - обыкновенный camelCase, ничем от верхних по неймингу не отличается, так как просто переменная

NazRudy replied:
Переделал

primus-anonymous commented on app/src/main/res/layout/activity_main.xml:
```diff
@@ -4,7 +4,206 @@
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <FrameLayout
```
Есть подозрение, что вот этот Frame тут уже лишний, те же отступы и background можно напрямую на TextView выставить

NazRudy replied:
Я выбрал низуий апи, и свойство backsgroung не работает, вышел из положения таким образом

primus-anonymous replied:
background вроде есть с самого первого андроида, я заинтригован

primus-anonymous commented on app/src/main/res/values/strings.xml:
```diff
@@ -1,3 +1,20 @@
     <string name="app_name">CalculatorHomeWork</string>
+    <string name="_0dis">0</string>
+    <string name="_1">1</string>
+    <string name="_2">2</string>
```
если используется camelCase, то key2, например, если snake, то key_2, начинать название с _ странно как-то.

NazRudy replied:
Исправил

primus-anonymous commented on app/src/main/res/values/strings.xml:
```diff
@@ -1,3 +1,20 @@
+    <string name="_4">4</string>
+    <string name="_5">5</string>
+    <string name="_6">6</string>
+    <string name="ADD">+</string>
```
add

NazRudy replied:
исправил

primus-anonymous commented on app/src/main/res/values/strings.xml:
```diff
@@ -1,3 +1,20 @@
+    <string name="_7">7</string>
+    <string name="_8">8</string>
+    <string name="_9">9</string>
+    <string name="SUB">-</string>
```
sub, dot и тд

NazRudy replied:
исправил

NazRudy committed:
Устранение замечаний
1. Реформат кода
2. Исправление названий переменных
3. По поводу двух фреймов, выбрал низкий апи, и свойство backsgrount не работает, вышел из положения таким образом.
